include/xrpl/protocol/detail/sfields.macro

```diff
@@ -136,6 +137,9 @@ TYPED_SFIELD(sfXChainClaimID,            UINT64,    20)
 TYPED_SFIELD(sfXChainAccountCreateCount, UINT64,    21)
 TYPED_SFIELD(sfXChainAccountClaimCount,  UINT64,    22)
 TYPED_SFIELD(sfAssetPrice,               UINT64,    23)
+TYPED_SFIELD(sfMaximumAmount,            UINT64,    24, SField::sMD_BaseTen|SField::sMD_Default)
```

Comment:
I don't think `MaximumAmount` can ever be `0` -- so shouldn't `Default` be removed?

Reply:
I believe setting sMD_Default only affects metadata generation of the transaction. But I dont know the metadata code well enough to give an answer to why we need this. cc @ximinez @gregtatcam

Reply:
`sMD_Default` is not identical to making a field `Default`. The flags provide some default behavior and are set by default when a field is constructed. Just setting the metadata to `sMD_BaseTen` overrides the default behavior and breaks the metadata generation for MPT amount fields.

Reply:
Ah, ok - thanks for explaining.